docs/UserGuide.md

```diff
@@ -483,27 +528,47 @@ A contact can have any number of tags. Add as many as you want.
 </div>
 
 <div markdown="block" class="alert alert-info">
-**:information_source: Note:** The tag has to be made first before you can add it into a contact.
+**:information_source: Note:** The tag has to be created first before you can add it into a contact.
+
+Refer to [`Creating a Tag`](#creating-a-tag-tag-create) on how to create a tag.
 </div>
 
-* Adds a `TAG` to the contact at the specified `INDEX`.
+**Example Input in Command Box**
+```
+tag add 1 t/friends
+```
 
-Example:
-* `tag add 1 t/friend` adds the friend tag to the first contact shown in the list.
+**Output in Command Result Box**
+```
+Tag added: [friends]
+```
+
+   (insert image of a successful addition of the `friends` tag to contact `1`)
+
+Awesome! You have successfully learnt to add a tag to a contact.
 
 <br>
 
 #### 7.4.5. Removing a Tag from a Contact: `tag remove`
 
-Removes an existing tag from an existing contact.
+You can remove a `TAG` from a contact.
 * `Coming soon in v1.5`, we will upgrade `tag remove` to remove tags from todos.
 
```

Comment:
Same as above, let the user knows how to remove tag from todos

Reply:
Solution is same as above